Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM centos:6
 WORKDIR /opt
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+ENV PERSEO_FE_URL=PERSEO_FE_ENDPOINT
```

Comment:
What about?

```
ENV PERSEO_FE_URL=perseo_fe_endpoint
```

Will achieve the same effect, but using lowercase for the env var value we are making more explicit that it refers to a literal (uppercase could be confused with the name of another env var, if the reader doesn't pay enough attention).

In this change is accepted, the same has to be done at https://github.com/telefonicaid/perseo-core/pull/73/files#diff-dc2504a86537f59b635ebc70c5237b54R3